waku/node/rest/health/client.nim

```diff
@@ -0,0 +1,30 @@
+when (NimMajor, NimMinor) < (1, 4):
```

Comment:
not sure if its too much to have another folder `health`? Would try to keep the minimum amount of folders. But yeah, not sure it fits in the existing ones.

Reply:
It currently fits with existing rest api structures. I would stick to it for now.

Reply:
sure, closing.